packages/flutter_tools/templates/app/README.md.tmpl

```diff
@@ -11,6 +11,6 @@ A few resources to get you started if this is your first Flutter project:
 - [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
 - [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
 
-For help getting started with Flutter, view our
+For help getting started with Flutter development, view the
 [online documentation](https://flutter.dev/docs), which offers tutorials,
```

Comment:
While you're right here, want to change this to `https://docs.flutter.dev`?

Reply:
Done.